docker-compose.multinode.yml

```diff
@@ -15,6 +15,7 @@ services:
       start_period: 10s
     volumes:
       - "${APPLICATION_ROOT_PATH}/config.multinode.txt:/opt/hgcapp/services-hedera/HapiApp2.0/config.txt"
+    network_mode: none
```

Comment:
this will cause the same issue as with the evm profile, you can test by running with 
`--multinode` flag

Reply:
@arianejasuwienas I don't see the changes, can you please double check if you've pushed all changes.
Regarding the no network being defined. The way the multinode mode is constructed is that `docker-compose.multinode.yml` is loaded on top of `docker-compose.yml` and overrides the values in there. Therefore the network defined in the base file (docker-compose.yml) are used. You can gain a deeper insight in how it all works by reviewing this function here https://github.com/hashgraph/hedera-local-node/blob/40357136639d9caeb3834404ed0d3b12ab37c16f/src/services/DockerService.ts#L327-L357 as all the logic around this is defined there.

Reply:
No, you're right, I've missed it before . Both

```
npm run start
```
and
```
npm run start --multinode 
```

seem to work correctly at this moment.

![Screenshot 2024-07-16 at 16 42 42](https://github.com/user-attachments/assets/1e978ade-c47b-41b6-a70d-9d4c48a49105)